src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        User newUser = new UserService().findByEmail(email);
```

Comment:
let's not create a new service each time we call a `login()` method, and make it an instance level variable of `AuthenticationService`
Also, this is basically not a new user, since we can say that he is already registered, so let's name this variable just `user`

Reply:
This comment is not resolved, you still create UserService each time when login methid is called